html/portal-connect.html

```diff
@@ -0,0 +1,231 @@
+              </div>
+            </div>
+
+            <!-- Nickname input -->
+            <div>
+              <!-- Nickname label -->
+              <div class="flex flex-row justify-between">
+                <label for="register-nickname-text" class="block text-sm font-medium text-gray-700">
+                  Nickname
+                </label>
+              </div>
+              <div class="mt-1">
+                <input id="register-nickname-text" name="nickname" type="text" autocomplete="nickname" required placeholder="Enter an identifying nickname for the account" class="appearance-none block w-full px-3 py-2 border border-gray-300 rounded-md shadow-sm placeholder-palette-300 focus:outline-none focus:ring-primary focus:border-primary text-xs sm:text-sm">
+              </div>
+            </div>
+
+            <!-- Register button -->
+            <div>
+              <button id="registere-submit" type="submit" class="w-full flex justify-center py-2 px-4 border border-transparent rounded-md shadow-sm text-sm font-medium text-palette-100 bg-primary hover:bg-primary-light transition-colors duration-200 focus:outline-none focus:ring-2 focus:ring-offset-2 focus:ring-primary disabled:opacity-50">
+                Sign Up
+              </button>
+            </div>
+          </form>
+
+          <!-- "back" link -->
+          <div class="mt-6 relative">
+            <div class="absolute inset-0 flex items-center">
+              <div class="w-full border-t border-gray-300"></div>
+            </div>
+            <div class="relative flex justify-center text-sm">
+              <span class="px-2 bg-white">
+                <a href="#" class="text-primary inline-flex items-center" onclick="window.goToInitialPage();">
+                  <svg xmlns="http://www.w3.org/2000/svg" class="h-4 w-4" fill="currentColor" class="bi bi-arrow-left-short" viewBox="0 0 16 16">
+                    <path fill-rule="evenodd" d="M12 8a.5.5 0 0 1-.5.5H5.707l2.147 2.146a.5.5 0 0 1-.708.708l-3-3a.5.5 0 0 1 0-.708l3-3a.5.5 0 1 1 .708.708L5.707 7.5H11.5a.5.5 0 0 1 .5.5z"/>
+                  </svg>
+                  <span id="register-back" class="ml-1">back</span>
+                </a>
+              </span>
+            </div>
+          </div>
+        </div>
+      </div>
+
+    </div>
+
+    <script>
+     const uiSigninConfirm = document.getElementById("signin-confirm");
+     const uiSigninSubmit = document.getElementById("signin-submit");
+
+     function confirmPortalAccountSignin() {
+       if (uiSigninConfirm.checked) {
+         uiSigninSubmit.removeAttribute("disabled")
+       } else {
+         uiSigninSubmit.setAttribute("disabled", "disabled")
```

Comment:
shouldn't the attribute's value be `true`?

Reply:
Looks like it's `disabled="disabled"`. Karol did this originally when he reworked the seed page and I assumed it was for a reason. 🤷‍♂️ Will follow up.

Reply:
Wrote this:
https://github.com/SkynetLabs/skynet-mysky/issues/273